addons/mail/static/tests/chatter_tests.js

```diff
@@ -197,16 +197,31 @@ QUnit.test('chatter is not rendered in mode === create', function (assert) {
     assert.strictEqual(form.$('.o_chatter').length, 1,
         "chatter should be displayed");
 
+    // entering create mode
     form.$buttons.find('.o_form_button_create').click();
+    assert.strictEqual(form.$('.o_chatter').length, 1,
+        "chatter should still be displayed in create mode");
 
-    assert.strictEqual(form.$('.o_chatter').length, 0,
-        "chatter should not be displayed");
+    // topbar buttons disabled in create mode (e.g. 'send message')
+    form.$('.o_chatter_topbar button').each(function () {
+        assert.ok($(this).is(":disabled"),
+            "button should be disabled in create mode");
+    });
 
+    // chatter containing a single message with 'Creating a record...'
+    assert.ok(form.$('.o_mail_thread').length, "there should be a mail thread");
+    assert.strictEqual(form.$('.o_thread_message').length, 1,
+        "there should be a single thread message");
+    assert.strictEqual(form.$('.o_thread_message_content').html().trim(),
```

Comment:
you could do something like
`form.$('.o_thread_message_content').text().trim()` to avoid useless `<p>` in your test